Example/ReusableDemo/TableViewController.swift

```diff
@@ -27,7 +27,8 @@ final class TableViewController: UITableViewController {
   }
 
   override func tableView(_ tableView: UITableView, viewForHeaderInSection section: Int) -> UIView? {
-    let view = MyHeaderTableView.loadFromNib()
+    let view = MyHeaderTableView(frame: CGRect(x: 0, y: 0, width: tableView.bounds.size.width, height: self.tableView(tableView, heightForHeaderInSection: section)))
+    view.loadNibContent()
```

Comment:
Maybe to better demonstrate the possibility of loading `NibOwnerLoadable` views from code, it would be better in this case to override the `init(frame: )` of `MyHeaderTableVew` to call `self.loadNibContent()` in that subclass' initializer… instead of having to think about calling it every time in the code we create a new `MyHeaderTableView` like here?